### ISSUE TITLE
No support even for so-called premium licence

### ISSUE THREAD

**Issue by j-steyn** [winmarkltd/BootstrapFormHelpers#265]
I paid for a 'premium' licence and downloaded the archive. Then the issues started...

Support link does not work, so it is not possible to raise a support request.

The Country Picker plugin does not play nicely with Bootstrap and does not allow a <select> element displaying country flags. As this was what I bought it for, WHAT A WASTE OF MONEY.

**Comment by lee-houghton** [comment]
If you paid with PayPal I'd suggest getting your money back though them. :expressionless:

**Comment by j-steyn** [comment]
Yeah, I'll do that. But it is still a sad state of affairs. :( And my form is still without a nice country picker with flags...
